src/app/containers/FrontPageSection/utilities/storySplitter.jsx

```diff
@@ -0,0 +1,41 @@
+import { splitEvery, splitAt, take } from 'ramda';
+
+export const getAllowedItems = (items, isFirstSection) =>
+  isFirstSection ? take(13, items) : take(10, items);
+
+// Split the top row stories out into their own list
+const splitFirstRow = (items, isFirstSection) => {
```

Comment:
Should make these more readable by having the following parts of this fuction in named constants.

Reply:
Done.